py-kms/KmsDataBase.xml

```diff
@@ -103,6 +103,10 @@
 			<Activate KmsItem="7ba0bf23-d0f5-4072-91d9-d55af5a481b6" />
 		</CsvlkItem>
 
+		<CsvlkItem DisplayName="Windows Server 2022" VlmcsdIndex="0" GroupId="206" MinKeyId="551000000" MaxKeyId="570999999" IniFileName="Windows" EPid="06401-00206-566-174993-03-1033-9600.0000-2802018" Id="ef6cfc9f-8c5d-44ac-9aad-de6a2ea0ae03" InvalidWinBuild="[0,1,2]">

```

Comment:
I am concerned by the clashing values for `GroupId` and `EPid` I have not tested this change and I am unsure on the success of @hyperboreanwolfgirl in their attempt to get this working.